src/syft/grid/example_nodes/network.py

```diff
@@ -20,33 +21,56 @@
 from syft.core.node.network.network import Network
 from syft.grid.services.signaling_service import PullSignalingService
 from syft.grid.services.signaling_service import PushSignalingService
+from syft.grid.services.signaling_service import RegisterDuetPeerService
 
 app = Flask(__name__)
 
 network = Network(name="om-net")
 
 network.immediate_services_without_reply.append(PushSignalingService)
 network.immediate_services_with_reply.append(PullSignalingService)
+network.immediate_services_with_reply.append(RegisterDuetPeerService)
 network._register_services()  # re-register all services including SignalingService
 
 
 @app.route("/metadata")
-def get_metadata() -> str:
-    return network.get_metadata_for_client()
+def get_metadata() -> flask.Response:
+    metadata = network.get_metadata_for_client()
+    metadata_proto = metadata.serialize()
+    r = Response(
+        response=metadata_proto.SerializeToString(),
+        status=200,
+    )
+    r.headers["Content-Type"] = "application/octet-stream"
+    return r
 
 
 @app.route("/", methods=["POST"])
-def process_network_msgs() -> str:
-    json_msg = request.get_json()
-    obj_msg = _deserialize(blob=json_msg, from_json=True)
+def process_network_msgs() -> flask.Response:
+    data = flask.request.get_data()
+    obj_msg = _deserialize(blob=data, from_bytes=True)
     if isinstance(obj_msg, SignedImmediateSyftMessageWithReply):
+        print(
```

Comment:
Q: Shouldn't we use ```log.info```?

Reply:
Good question, this is the CLI service running on the web server and we will be adding a Docker container for people to easily deploy, so for ease of debugging printing out to the stdout is probably helpful. But there might be some preferred logging configuration to ensure log retention and auditing of connections within certain networks. For now I guess this is a lightweight example so perhaps this should be addressed as part of Grid?